src/components/AttachmentPicker/index.native.js

```diff
@@ -88,160 +85,61 @@ function getDataForUpload(fileData) {
-                pickAttachment: () => this.showImagePicker(launchImageLibrary),
-            },
-        ];
-
-        // When selecting an image on a native device, it would be redundant to have a second option for choosing a document,
-        // so it is excluded in this case.
-        if (this.props.type !== CONST.ATTACHMENT_PICKER_TYPE.IMAGE) {
-            this.menuItemData.push({
-                icon: Expensicons.Paperclip,
-                textTranslationKey: 'attachmentPicker.chooseDocument',
-                pickAttachment: () => this.showDocumentPicker(),
-            });
-        }
-
-        this.close = this.close.bind(this);
-        this.pickAttachment = this.pickAttachment.bind(this);
-        this.removeKeyboardListener = this.removeKeyboardListener.bind(this);
-        this.attachKeyboardListener = this.attachKeyboardListener.bind(this);
-    }
-
-    componentDidUpdate(prevState) {
-        if (this.state.isVisible === prevState.isVisible) {
-            return;
-        }
-
-        if (this.state.isVisible) {
-            this.attachKeyboardListener();
-        } else {
-            this.removeKeyboardListener();
-        }
-    }
-
-    componentWillUnmount() {
-        this.removeKeyboardListener();
-    }
-
-    attachKeyboardListener() {
-        const shortcutConfig = CONST.KEYBOARD_SHORTCUTS.ENTER;
-        this.unsubscribeEnterKey = KeyboardShortcut.subscribe(
-            shortcutConfig.shortcutKey,
-            () => {
-                if (this.state.focusedIndex === -1) {
-                    return;
-                }
-                this.selectItem(this.menuItemData[this.state.focusedIndex]);
-                this.setState({focusedIndex: -1}); // Reset the focusedIndex on selecting any menu
-            },
-            shortcutConfig.descriptionKey,
-            shortcutConfig.modifiers,
-            true,
-        );
-    }
-
-    removeKeyboardListener() {
-        if (!this.unsubscribeEnterKey) {
-            return;
-        }
-        this.unsubscribeEnterKey();
-    }
-
-    /**
-     * Handles the image/document picker result and
-     * sends the selected attachment to the caller (parent component)
-     *
-     * @param {Array<ImagePickerResponse|DocumentPickerResponse>} attachments
-     * @returns {Promise}
-     */
-    pickAttachment(attachments = []) {
-        if (attachments.length === 0) {
-            return;
-        }
-
-        const fileData = _.first(attachments);
-
-        if (fileData.width === -1 || fileData.height === -1) {
-            this.showImageCorruptionAlert();
-            return;
-        }
-
-        return getDataForUpload(fileData)
-            .then((result) => {
-                this.completeAttachmentSelection(result);
-            })
-            .catch((error) => {
-                this.showGeneralAlert(error.message);
-                throw error;
-            });
-    }
+function AttachmentPicker(props) {
```

Comment:
Feel free to use the destruction assignment for props.
```suggestion
function AttachmentPicker({...}) {
```

Reply:
Done!